### ISSUE TITLE
Verbose Feature on MWEM No Longer Compatible with Python 3.7

### ISSUE THREAD

**Issue by GenericP3rson** [opendp/smartnoise-sdk#526]
Running smartnoise-synth==0.3.5. 
Python Version: Python 3.7.0

It looks like MWEM is no longer compatible with Python 3.7 when the verbose function is set to true (even though the README indicates the package is compatible with Python 3.7). 

Code (slightly modified from from the [docs](https://docs.smartnoise.org/synth/index.html)):

```
from snsynth import Synthesizer
import pandas as pd

pums = pd.read_csv('PUMS.csv', index_col=None) # in datasets
pums = pums.drop(['income', 'age'], axis=1)

synth = Synthesizer.create('mwem', epsilon=1.0, verbose=True)
sample = synth.fit_sample(pums)
print(sample)
```

Error:
```
AttributeError: module 'math' has no attribute 'comb'
```

This error is because "comb" is a new method in [Python 3.8](https://docs.python.org/3.8/library/math.html) that does not exist in [Python 3.7](https://docs.python.org/3.7/library/math.html).

The function works without the `verbose=True` parameter or with Python 3.8.

**Comment by joshua-oss** [comment]
Thanks for reporting and pinpointing the root cause.  The bug is fixed and pushed to `smartnoise-synth==3.5.1` on PyPi.

**Comment by GenericP3rson** [comment]
Awesome, it's working now! Thank you so much!
